dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1662,7 +1662,7 @@ a.download-video {
   overflow: hidden;
 }
 
-.text-match, .free-response {
```

Comment:
I was asked to move PLC stuff to code-studio a few months ago but don't remember the reasoning